test/test_vocab.py

```diff
@@ -109,7 +108,7 @@ def test_save_load(self):
         vocab_loaded = vocab.Vocabulary.load(fname)
 
         self.assertEqual(len(self.vocab), len(vocab_loaded))
-        for k in xrange(len(self.vocab)):
+        for k in range(len(self.vocab)):
```

Comment:
It's better to convert these to use either `six.moves.range` or futures `builtins.range` rather than reverting these to use `range` on Python 2 since that materializes a large array.

Reply:
Actually, these are all just in the test, so I don't care about the performance there.